src/crypto.rs

```diff
@@ -78,9 +80,7 @@ impl Default for Algorithm {
     }
 }
 
-pub fn get_pseudonymizer(algo: Option<Algorithm>, key: Option<Vec<u8>>) -> impl Pseudonymize {
-    let secret = key.unwrap_or(generate_key(32));
-
+pub fn get_pseudonymizer(algo: Option<Algorithm>, secret: Option<Vec<u8>>) -> impl Pseudonymize {
```

Comment:
Maybe this is not relevant for this PR, but shouldn't we rename this into `new_pseudonymizer` or use a method of `Pseudonymizer` instead (`Pseudonymizer::new`)?
The function creates a new pseudonymizer, it does not really "get" it.